pypeit/specobjs.py

```diff
@@ -100,8 +89,8 @@ def __init__(self, shape, slit_spat_pos, slit_spec_pos, det=1, setup=None, idx =
         self.trace_spat = None
         self.spat_pixpos = spat_pixpos # Position on the image in pixels at the midpoint of the slit in spectral direction
         self.maskwidth = None
-        self.mincol = None
-        self.maxcol = None
+        self.min_spat = None
```

Comment:
define this in `Attributes:`

Reply:
I think we discussed that this whole objects would be revamped so I'll hold off on that for now.